aten/src/ATen/native/native_functions.yaml

```diff
@@ -14611,3 +14611,11 @@
   dispatch:
     CUDA: _fused_adamw_kernel_cuda_
   autogen: _fused_adamw, _fused_adamw.out
+
+# Collectives
+- func: all_reduce(Tensor self, str reduceOp, str tag, int[] ranks, int stride) -> Tensor
+  # This should be changed to distributed but it requires changes all over the place to work
+  python_module: nn
```

Comment:
Oh this is why we are getting the `nn` namespace: `torch._C._nn.all_reduce`, is this required bc we are registering a python op?